listpokemon/src/main/java/com/bsarias/pokeapi/list/usecases/GetListPokemons.kt

```diff
@@ -0,0 +1,9 @@
+package com.bsarias.pokeapi.list.usecases
+
+import com.bsarias.pokeapi.core.domain.ListPokemon
+import com.bsarias.pokeapi.list.data.repository.ListPokemonRepository
+
+class GetListPokemons(private val listPokemonRepository: ListPokemonRepository) {
+    suspend fun invoke(limit: Int, offset: Int, key: String): ListPokemon =
```

Comment:
```suggestion
    suspend operator fun invoke(limit: Int, offset: Int, key: String): ListPokemon =
```
This way you can use `getListPokemons(...)` instead of `getListPokemons.invoke(...)`